taxi/views.py

```diff
@@ -1,47 +1,55 @@
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
 from django.shortcuts import render
 from django.views import generic
 
+
 from .models import Driver, Car, Manufacturer
 
 
+@login_required
 def index(request):
```

Comment:
add typehint for function parameters and return value

Reply:
Alright, thank you.